openhabian-setup.sh

```diff
@@ -10,19 +10,6 @@
 #
 
 
-# Find the absolute script location dir (e.g. BASEDIR=/opt/openhabian)
-SOURCE="${BASH_SOURCE[0]}"
-while [ -h "$SOURCE" ]; do
-  BASEDIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
-  SOURCE="$(readlink "$SOURCE")"
-  [[ $SOURCE != /* ]] && SOURCE="$BASEDIR/$SOURCE"
-done
-BASEDIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
-SCRIPTNAME="$(basename "$SOURCE")"
```

Comment:
back to this implementation w/o init.bash?

Reply:
yes